docs/releases/v2022.1.4.rst

```diff
@@ -0,0 +1,126 @@
+Gluon 2022.1.4
+==============
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- LibreRouter
+
+  - LibreRouter (v1)
+
+- Teltonika
+
+  - RUT230 (v1)
+
+
+ath79-nand
+~~~~~~~~~~
+
+- NETGEAR
+
+  - WNDR4300 (v1)
+  
```

Comment:
```suggestion

```

line contains two spaces. I'm not arguing to remove the line, but the spaces. GitHubs UI is weird regarding empty lines.